packages/frontend/src/app/page.tsx

```diff
@@ -1,113 +1,50 @@
-import Image from 'next/image'
+import React from 'react'
+import Button from '@/common/components/Button'
+import Table, { TableDataConfigItem } from '@/common/components/Table'
+import PageHeading from '@/common/components/PageHeading'
```

Comment:
Could we do named export for `PageHeading` as with other components?

Reply:
Same also for `Button`